components/Resources/ResourcesSearchResults.vue

```diff
@@ -12,10 +12,10 @@
         <a :href="data.fields.url" target="blank">
           <h2>{{ data.fields.name }}</h2>
         </a>
-        <p class="resources-search-results__items--content-date">
-          {{ formatDate(data.sys.updatedAt) }}
-        </p>
 
+        <template v-if="data.fields.developedBySparc">
+          <p class="resource-category">SPARC</p>
+        </template>
```

Comment:
nit: No `template` needed.
```suggestion
        <p v-if="data.fields.developedBySparc" class="resource-category">SPARC</p>
```